cloudify_marketplace/provision/check_prerequisites

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+MIN_RAM_GB=3
```

Comment:
shouldn't these match the validations we perform during bootstrap?

currently, it's set for 4GB RAM, 5GB HD..